.babelrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Storybook now requires a babel config, this is autogenerated to be compatible with what 6.x provided